helm/aws-operator-chart/templates/03-configmap.yaml

```diff
@@ -11,6 +11,9 @@ data:
     service:
       aws:
         region: '{{ .Values.Installation.V1.Provider.AWS.Region }}'
+      guest:
+        update:
+          enabled: {{ .Values.Installation.V1.Guest.Update.Enabled }}
```

Comment:
The default as of now is `false`, which means that, if not configured otherwise, updates are not allowed anywhere, but `geckon`. There will be PRs in the installations repo to enable updates in e.g. `gauss` in order to test further update implementations.